scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -594,7 +594,7 @@
              (take num-considerable)
              ; Force this to be taken eagerly so that the log line is accurate.
              (doall))]
-    (swap! pool->user->number-jobs update pool-name (constantly @user->number-jobs))
+    (swap! pool->user->number-jobs update pool-name (constantly @user->rate-limit-count))
```

Comment:
Can we please rename `pool->user->number-jobs` to `pool->user->num-rate-limited-jobs`?

Reply:
Done, good call